assets/css/main.css

```diff
@@ -447,3 +447,30 @@ table.results-table .category-row {
     margin-top: 4rem;
     width: 80%;
 }
+
+/* Skip link stuff */
```

Comment:
Is the best location for these in the end of the file? I can be, but often people just add it to the end and then the content of the file will have no sorting after a while. To be fair, the file has a lot of that already, thus finding a good place is hard.

I typically see first generic stuff, then definitions matching elements in all pages and starting from the top of the page. Lastly, I would expect to find items on rare pages and often would hope those to be in different files. As this exists in few places in the page, would it be more suitable somewhere around lines 200-240 maybe?

Reply:
Without wishing to sound like a stuck record, with better tooling we could use one file per component and squish and minify them on deployment, which would be most logically organised and efficient.

I've never navigated big CSS files any other way than searching for selectors, so I'll yield to your view on this.

Reply:
> Without wishing to sound like a stuck record, with better tooling we could use one file per component and squish and minify them on deployment, which would be most logically organised and efficient.

Yes. This is the reason why I would hope we could to do that (relatively simply considering current deployment methods). For now, if the content applies to only few pages (e.g. login), then it's good to use a dedicated file. Due to HTTP/2 and 3, there isn't so much need for bundling the files anymore. Sadly, all minify tools seem to do both and configuring them to do only minifying is hard or complicated. I was planning on writing a new library to be run with `./manage.py collectstatic`, which would download remote files, minify local files and fix internal links, but I haven't had time for that yet.

> I've never navigated big CSS files any other way than searching for selectors, so I'll yield to your view on this.

I try to not to repeat past mistakes, so thank you. Personally, I consider cleaning up stuff to be one of the hardest things in the information technology.